### PR TITLE
NextJS integration

### PR DESCRIPTION
**Pull request by ilfa**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ The main branch is locked for the push action. For proposing changes, use the st
 
 ### Development playground
 
-It's a demo page in `/examples/spa` folder. You can find more info about configuration and starting demo in [readme](examples/spa/README.md).
+It's a 2 demo page for this integration:
```
Comment (TheUnderScorer) [inline contributing.md:11]: How about:

```suggestion
There are 2 demo pages for this integration:
```

?

File: examples/next/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+## Setting up
+In order to try out these examples, you set your public API key as the value of the NEXT_PUBLIC_FPJS_PUBLIC_API_KEY variable\
```
Comment (TheUnderScorer) [inline examples/next/README.md:2]:
```suggestion
In order to try out these examples, you need to set your public API key as the value of the NEXT_PUBLIC_FPJS_PUBLIC_API_KEY variable\
```

**Review by TheUnderScorer — APPROVED** [review]

File: examples/next/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+## Setting up
+In order to try out these examples, you need to set your public API key as the value of the NEXT_PUBLIC_FPJS_PUBLIC_API_KEY variable\
+in the `.local.env`, you can just copy `.env` file, located in this directory.
+
+### FingerprintJS public API key
```
Comment (makma) [inline examples/next/README.md:5]: I'd remove this section. Whenever it's possible try to link our official docu. I'd use this link here - https://dev.fingerprintjs.com/docs/js-agent#agent-initialization. It'll be easier to maintain.

File: examples/next/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+## Setting up
+In order to try out these examples, you need to set your public API key as the value of the NEXT_PUBLIC_FPJS_PUBLIC_API_KEY variable\
```
Comment (makma) [inline examples/next/README.md:2]:
```suggestion
To try out these examples, you need to set your public API key as the value of the NEXT_PUBLIC_FPJS_PUBLIC_API_KEY variable\
```

File: examples/next/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+in the `.local.env`, you can just copy `.env` file, located in this directory.
```
Comment (makma) [inline examples/next/README.md:3]:
```suggestion
in the `.local.env`. You can just copy `.env` file located in this directory.
```

File: examples/next/README.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+### FingerprintJS public API key
+- Go to [FingerprintJS Dashboard](https://dashboard.fingerprintjs.com/)*
+- Open the _API keys_ page from the sidebar
+- Find your _Public_ API key
+
+*If you don't have a FingerprintJS Pro account, [sign up for free](https://dashboard.fingerprintjs.com/signup/)).
+
+### Installing dependencies
+1. Go to the root of the project, install dependencies and build it (`cd ../../ && yarn install && yarn build`)
```
Comment (makma) [inline examples/next/README.md:13]: Wouldn't it better to create these scripts in the package.json ant then ask them to run just one or two commands?

**Review by makma — CHANGES_REQUESTED** [review]
Before merging pls ask somebody on the team for testing/feedback from the developer PoV.

File: examples/next/README.md (hunk 3 above)
Comment (ilfa) [reply examples/next/README.md]: done

File: examples/next/README.md (hunk 6 above)
Comment (ilfa) [reply examples/next/README.md]: I suppose it is overengineering to create a separate script. Rewrote the text for better perception.

**Review by TheUnderScorer — APPROVED** [review]

**Review by makma — APPROVED** [review]

**Pull request merged**